lib/pc_util.c

```diff
@@ -156,6 +156,30 @@ wkb_get_int16(const uint8_t *wkb, int flip_endian)
 		return i;
 }
 
+uint8_t *
+wkb_set_double(uint8_t *wkb, double d)
+{
+	memcpy(wkb, &d, 8);
+	wkb += 8;
```

Comment:
I agree that it's better to use `sizeof` but the `wkb_get_*` functions and other functions in this file use numerical values instead of `sizeof`. I'd rather favor consistency and use numerical values here in the setters as well.